docs/openshift.md

```diff
@@ -101,6 +122,10 @@ Verify that the deployment runs successfully using the following commands.
 oc get pods
 ```
 
+### Deploying OpenEBS Jiva using NodeSelector.
+By using the NodeSelector approach the OpenEBS Jiva can be also deployed. Suppose you have 5 nodes and you want to use 2 nodes for application and 3 nodes for storage. This requirement can be achieved by using NodeSelector. The detailed have been mentioned in this link. [NodeSelector](/docs/next/scheduler.html)
```

Comment:
By using the NodeSelector approach, OpenEBS Jiva...
Suppose you have = For example, if you have...3 nodes for storage, you can achieve this using NodeSelector. 
The detailed have been = Details are mentioned in this link [...].